book/02-using-atom/sections/01-packages-themes.asc

```diff
@@ -1,7 +1,7 @@
 [[_atom_packages]]
 === Atom Packages
 
-First we'll start with the Atom package system. As we mentioned previously, Atom itself is a very basic core of functionality that ships with a number of useful packages that add new features like the Tree view and the Settings view (which, by the way, can be found here: https://github.com/atom/settings-view).
+First we'll start with the Atom package system. As we mentioned previously, Atom itself is a very basic core of functionality that ships with a number of useful packages that add new features like the Tree view and the Settings view (which, by the way, can be found here: https://github.com/atom/settings-view[https://github.com/atom/settings-view]).
```

Comment:
What do you think about using `Settings view` for the link, and deleting the rest? So, something like: 

>  ... add new features like the https://github.com/atom/tree-view[Tree view] and the https://github.com/atom/settings-view[Settings view].